load-balancer/src/main/java/org/apache/phoenix/loadbalancer/service/LoadBalanceZookeeperConfImpl.java

```diff
@@ -65,8 +65,23 @@ public String getServiceName(){
 
         @Override
         public String getZkConnectString(){
-            return String.format("%s:%s",configuration.get(QueryServerProperties.ZOOKEEPER_QUORUM_ATTRIB,
-                    "localhost"),configuration.get(QueryServerProperties.ZOOKEEPER_PORT_ATTRIB,"2181"));
+            String[] serverHosts = configuration.getStrings(QueryServerProperties.ZOOKEEPER_QUORUM_ATTRIB, "localhost");
+            String clientPort = configuration.get(QueryServerProperties.ZOOKEEPER_PORT_ATTRIB, "2181");
+            StringBuilder zkConnectionString = new StringBuilder();
```

Comment:
Why not just call ZKConfig.standardizeZKQuorumServerString() from HBase?
We are pulling in hbase-common anyways.